doc/source/user-guide/linkcode.rst

```diff
@@ -1,20 +1,11 @@
 .. _ref_user_guide_extension:
 
-Extensions
-==========
-
-This page contains the shipped extensions that seamlessly integrate with this theme to enhance
-documentation functionality.
-
 Linkcode extension
--------------------
+===================
 
 The Linkcode extension automatically adds "Source" links to the documentation for Python, C, C++,
 and JavaScript objects. It allows you to link to the source code hosted on GitHub.
 
-Installation
-~~~~~~~~~~~~
-
 To use the ``ansys-sphinx-theme`` linkcode extension, you need to add it to the Sphinx configuration:
 
 #. Add the following line in the ``conf.py`` file:
```

Comment:
```suggestion
To use the ``linkcode`` extension, you must add it to the ``extensions`` variable in your
project's Sphinx ``conf.py`` file:
```
You shouldn't have a procedure with only one step. You will need to fix the indentation of the code block.